### ISSUE TITLE
plote ROPE analysis with no intercepts?

### ISSUE THREAD

**Issue by dontchaknowjo** [easystats/bayestestR#556]
**Question and context**
Hello, I was wondering if anyone has ideas about how to plot the ROPE analysis without plotting the intercepts. 

here is what I tried that did not work:

percentage_in_rope1 <- rope(fit_sd, range=c(-0.1,0.1),ci=1)

percentage_in_rope1_2<- dplyr::filter(percentage_in_rope1, Parameter == "b_Hierarchy_ZeroSum")

plot(percentage_in_rope1_2)

**Comment by dontchaknowjo** [comment]
I also discovered that you can call show_intercept=FALSE, but this isn't affecting the plots of my rope analysis.

**Comment by strengejacke** [comment]
> @strengejacke Could we port the `keep` and `drop` arguments from _parameters_ to the various _bayestestR_ functions?

Yes, we could do that. However, `show_intercept = FALSE` should be available, there seems to be a bug in the `plot()` method. @dontchaknowjo Do you have a reproducible example?

**Comment by dontchaknowjo** [comment]
Yes thanks for asking. Let me know if you need anything else from me!



fit_interact_imp<- brm(important_manuscripts ~ Hierarchy_ZeroSum*self_MS_cooper+Identify+LogYears,
                       data=TEST_data_S2,
                       family=cumulative(probit),
                       save_all_pars = TRUE,
                       iter= 6000, warmup= 1000,
                       thin= 1,
                       inits = "0",
                       prior=priors)



plot(rope(fit_interact),range=c(-0.1,0.1),ci=1,show_intercept=FALSE)+
  ggplot2::labs(title = "Do you plan to share your next manuscript?")+xlim(-1,1)
